projects/manager/src/app/providers/provider-editor/forms/aruna/aruna.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-card>
+    <mat-card-content [formGroup]="form">
+        <geoengine-manager-id-input *ngIf="createNew" [id]="idControl"></geoengine-manager-id-input>
```

Comment:
where do you actually use this input?

Reply:
When adding a new provider in the Aruna form. It only makes sense for a new provider since it should not be possible to change an existing ID.